sdk/paloaltonetworks/azure-resourcemanager-paloaltonetworks-ngfw/CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+# Release History
+
+## 1.0.0-beta.1 (2023-04-17)
+
+- Azure Resource Manager ngfw client library for Java. This package contains Microsoft Azure SDK for ngfw Management SDK.  Package tag package-2022-08-29-preview. For documentation on how to use this package, please see [Azure Management Libraries for Java](https://aka.ms/azsdk/java/mgmt).
```

Comment:
`Azure Resourcemanager PaloAlto Networks ngfw client library for Java`?

Reply:
Please clean up the 2nd line.